interceptors/src/unit-test/java/uk/gov/moj/sdt/interceptors/in/ServiceRequestInboundInterceptorTest.java

```diff
@@ -67,17 +67,20 @@ class ServiceRequestInboundInterceptorTest extends AbstractSdtUnitTestBase {
     @Mock
     ServiceRequestDao mockServiceRequestDao;
 
+    RequestDaoService requestDaoService;
+
     /**
      * Test that the process correctly works via mocked out extensions.
      */
     @Test
     void testHandleMessage() {
```

Comment:
Looks like the only thing that can be checked in testHandleMessage() is to verify that persist() has been called on the mocked ServiceRequestDao, i.e.
`Mockito.verify(mockServiceRequestDao).persist(Mockito.any(IServiceRequest.class));`.  Can use static imports instead of prefixing the methods with Mockito.